dist/demo.html

```diff
@@ -1,8 +0,0 @@
-<meta charset="utf-8">
```

Comment:
I'd revert `dist/` to the original state, just to avoid the noise, because I don't think it matters in the end.

Reply:
[`1042680` (#99)](https://github.com/bethinkpl/design-system/pull/99/commits/1042680e5eff8e919f7771ec1abe21bff706a7a6)